db/system_keyspace.cc

```diff
@@ -99,6 +99,33 @@ namespace {
             props.enable_schema_commitlog();
         }
     });
+
+    const auto set_group0_table_options =
+        schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
+            static const std::unordered_set<sstring> tables = {
+                // scylla_local may store a replicated tombstone related to schema
+                // (see `make_group0_schema_version_mutation`), so we the repair GC mode
```

Comment:
"so we the repair GC mode" <- so we use?

Reply:
fixed